src/app/HorizontalNav/HorizontalNav.tsx

```diff
@@ -1,72 +1,169 @@
-import React from 'react';
+import React, { useState, useEffect } from "react";
 import {
-  Nav, TextContent,Avatar,
-  Text, NavExpandable, TextVariants,NavItem, NavItemSeparator, NavList, NavGroup, PageHeader
-} from '@patternfly/react-core';
-const avatarImg = "https://www.patternfly.org/v4/images/avatarImg.668560cdf25a4932ef9f711b4acad52d.svg";
-
+  Popover,
+  Card,
+  CardBody,
+  Nav,
+  ProgressStepper,
+  TextContent,
+  Avatar,
+  Text,
+  ProgressStep,
+  TextVariants,
+  NavItem,
+  NavList,
+} from "@patternfly/react-core";
+import { get_ip, get_port } from "env";
 
 const HorizontalNav: React.FunctionComponent = () => {
+  const [activeItem, setActiveItem] = useState(0);
+  const [avtarImg, setAvtarImg] = useState("");
+  const [autotuneOn, setAutotuneOn] = useState("");
+  const ip = get_ip();
+  const port = get_port();
 
-  const [activeItem, setActiveItem] = React.useState(0);
-  const onSelect = result => {
-    setActiveItem(result)
-  }
-  const nav = (
+  useEffect(() => {
+    
+     if( ip ){
+        setAvtarImg("https://upload.wikimedia.org/wikipedia/labs/b/ba/Kubernetes-icon-color.svg")
+     }
+    else{
+          setAvtarImg("https://www.patternfly.org/v4/images/avatarImg.668560cdf25a4932ef9f711b4acad52d.svg")
+        }
+      },[]);
+
+  useEffect(() => {
+    if(ip && port)
+      {
+        setAutotuneOn(ip+port)
```

Comment:
This can be a set with a `boolean` rather than ip and port. Please check if it's `true` or `false` when you want to check the autotune